scripts/publish.sh

```diff
@@ -118,6 +118,20 @@ publish_to_docker_hub() {
 
 }
 
+publish_to_public_ecr() {
+	aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
```

Comment:
for testing, use a different public repo. Create one in your own account and make the code publish to it.